ggml.c

```diff
@@ -15763,8 +15525,17 @@ static void ggml_compute_forward_flash_attn_ext_f16(
     const int ir0 = dr*ith;
     const int ir1 = MIN(ir0 + dr, nr);
 
-    float scale = 1.0f;
-    memcpy(&scale, (float *) dst->op_params + 0, sizeof(float));
+    float scale    = 1.0f;
+    float max_bias = 0.0f;
+
+    memcpy(&scale,    (float *) dst->op_params + 0, sizeof(float));
```

Comment:
just to confirm, this is needed because to set params we also do a `memcpy` instead of making `op_params` float[] for alignment reasons right?

Reply:
Yes, correct